index.js

```diff
@@ -1,60 +1,29 @@
-const {app, BrowserWindow, ipcMain, Menu, shell} = require('electron')
 const defaultMenu = require('electron-default-menu')
+const window = require('electron-window')
+const electron = require('electron')
+const Env = require('envobj')
+const path = require('path')
 
-const env = process.env.NODE_ENV
+const delegateEvents = require('./lib/delegate-electron-events')
 
-let win, file, link
+const shell = electron.shell
+const Menu = electron.Menu
+const app = electron.app
```

Comment:
Oh, I didn't consciously change it, sorry about that - I just rewrote this file from scratch and incrementally added requires as I needed them like I normally do.

That said I do dislike multiple variable declarations on a single line as I find them hard to read and modify. I believe in most of our code we aren't using this, so only using it in this particular case feels like an odd one out.

I think most people on the team (myself included) might not necessarily be familiar with how destructuring works in detail. E.g. saw this in a tweet the other day, had no idea what was going on 😕:

[![cxtvcqwwqaawkl3](https://cloud.githubusercontent.com/assets/2467194/20497478/964bb36c-b029-11e6-89c7-270a5f7cf824.jpg)](https://twitter.com/WickyNilliams/status/798497274653503488)

But yeah guess since you seem to feel strongly about this I'll add them back; I'd rather not get stuck arguing syntax haha :sparkles: